### ISSUE TITLE
Support cooldowns

### ISSUE THREAD

**Issue by XuaTheGrate** [XuaTheGrate/slash_util#6]
Edit: #14 has added support for checks. cooldowns still required.

**Comment by ChesterWOV-DPY** [comment]
I don't think we could make the `discord.ext.commands` checks work for slash_util tbh, I'm thinking that we will have to make `@check` ourselves
